app/Http/Transformers/PhoenixGalleryTransformer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Post;
+use League\Fractal\TransformerAbstract;
+
+class PhoenixGalleryTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param \Rogue\Models\Photo $photo
+     * @return array
+     */
+    public function transform(Post $post)
+    {
+        $signup = $post->signup;
+        $result = [
+            'id' => $post->postable_id,
+            'status' => $post->status,
+            'caption' => $post->content->caption,
+            'uri' => $post->content->file_url,
```

Comment:
Same here. Also I don't think we need to return it twice in the response object.